CHANGELOG.md

```diff
@@ -10,6 +10,16 @@ and this project adheres to
 
 ### Added
 
+### Changed
+
+### Deprecated
+
+### Fixed
+
+## \[1.8.0\]
```

Comment:
Just to double check: None of the entries below were added after the 1.8 release branch was cut, right? (Apart from the MSR thing, which got backported)

Reply:
Correect